src/components/select/select-theme.scss

```diff
@@ -75,3 +75,17 @@ md-select-menu.md-THEME_NAME-theme {
     }
   }
 }
+
+md-select-menu[multiple] {
+    md-option {
```

Comment:
ahh apologies, my vimrc did not recognize scss and defaulted to 4 spaces. Corrected.